app/src/test/java/com/github/se/eventradar/ChatViewModelUnitTest.kt

```diff
@@ -167,14 +167,11 @@ class ChatViewModelUnitTest {
     val msg1 = mockMessage.copy(sender = "user1", id = "msg1")
     val msg2 = mockMessage.copy(sender = "user2", id = "msg2")
 
-    // TODO: clean up logic with add message
-    val messageHistory = messageRepository.createNewMessageHistory("user1", "user2")
+    val resource = messageRepository.createNewMessageHistory("user1", "user2")
+    val messageHistory = (resource as Resource.Success).data
 
-    val editedMH = (messageHistory as Resource.Success).data
-
-    editedMH.messages.add(msg1)
-
-    messageRepository.addMessage(msg2, editedMH)
+    messageRepository.addMessage(msg1, messageHistory)
+    messageRepository.addMessage(msg2, messageHistory)
 
     viewModel =
         ChatViewModel(messageRepository, userRepository, opponentId) // Initialize view model
```

Comment:
Nit: the line below: ` runBlocking { viewModel.getMessages() }` is not necessary as it already called upon init.